documentation/src/main/asciidoc/introduction/Advanced.adoc

```diff
@@ -994,7 +994,9 @@ plugins {
     id "org.hibernate.orm" version "{fullVersion}"
 }
 
-hibernate { enhancement }
+hibernate {
+  enhancement {}
+}
 ----
```

Comment:
Maybe add this?

```suggestion
----

[CAUTION]
====
Some online documentation (including previous versions of the present one) suggest to use `hibernate { enhancement }`, which will _not_ work as it is interpreted by Gradle as a (pointless) getter call instead of actual configuration. 

Make sure you always pass `{}` to the `enhancement` call.
====
```